user-guide/Reference/faq/IDs_of_views_elements_services_redundancy_groups_and_parameters.md

```diff
@@ -25,11 +25,11 @@ To find this ID in DataMiner Cube:
 
 1. Open the view containing the element, service or redundancy group.
 
-1. On the view card, go to the data side.
+1. Under BELOW THIS VIEW there is a page for each type of item that can be contained within the view. Using the ALL page will display all items within the view.
 
-1. Click the *details* tab.
+    Next to the item name, a number is displayed that indicates how many of these items are in the view. Click a page in the tree view to display a list of the items in question.
 
-   The IDs of all items in the view can be found in the *ID* column.
+    The IDs of all items in the view can be found in the *ID* column.
```

Comment:
Change this to a second step:
1. Check the *ID* column to find the ID of the element, service, or redundancy group.